tests/test-spawn.lua

```diff
@@ -86,13 +87,24 @@ local steps = {
                                          exit_snd = code
                                      end
                                  })
+            spawn.easy_async("ping -c1 localhost", function(stdout, stderr, reason, exit_code)
+                if stdout:match("0%% packet loss") then
+                    async_spawns_done = async_spawns_done + 1
+                end
+            end)
+            spawn.easy_async_with_shell("ping -c1 localhost | grep 'packet loss' | cut -d' ' -f6", function(stdout, stderr, reason, exit_code)
+                if stdout:match("0%%") then
+                    async_spawns_done = async_spawns_done + 1
+                end
+            end)
         end
         if spawns_done == 2 then
```

Comment:
i think the current test results (4/5 pass) are caused by exactly this problem

https://travis-ci.org/awesomeWM/awesome/builds/211464746